Web.Dependencies.Javascript/Web.Dependencies.Javascript.csproj

```
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```
I think we should use netstandard2.0 here to prevent us needing to update this project file each year.